Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -171,6 +205,32 @@ public async Task<ParticipantExtendedViewModel> PostParticipantAsync(string user
             return participant;
         }
 
+        // change to ParticipantUploadViewModel: a new viewmodel only for adds custom participants
+        public async Task<ParticipantExtendedViewModel> PostCustomParticipantAsync(string username, CustomParticipantViewModel newCustomParticipant)
+        {
+            var newUsername = GetCustomUnqiueUsername(username);
+
+            await _context.Participant.AddAsync(new Participant
+            {
+                Username = newUsername,
+                SpecifiedGender = newCustomParticipant.SpecifiedGender ?? "U",
+                IsCustom = true,
+                AllowEmails = newCustomParticipant.AllowEmails ?? true,
+                FirstName = newCustomParticipant.FirstName,
+                LastName = newCustomParticipant.LastName,
+            });
```

Comment:
I mentioned this in the UI PR, but we don't have an email for the user. Either you need to rework where we store emails. OR you need to default AllowEmails to false.

Reply:
Yes, I am thinking to rework the email storing. Even though we don't send an invite request to custom participants, but a notification email would still be nice to have.

Reply:
right so we would need a new column is what im saying. otherwise this needs to be set to false.